projects/ng-components-docs/src/app/examples/components/radio-group/examples/radio-groups/radio-groups.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from "@angular/core";
+import { Item } from "lib/src/radio-group/radio-group.component";
+
+@Component({
+    selector: "app-radio-groups",
+    templateUrl: "./radio-groups.component.html",
+})
+export class RadioGroupsComponent implements OnInit {
+    constructor() {}
+    list: Item[] = [
+        { key: "One", value: "1", label: "One" },
+        { key: "Two", value: "2", label: "Two" },
+    ];
+
+    ngOnInit() {}
+}
```

Comment:
```suggestion
import { Component } from "@angular/core";
import { Item } from "lib/src/radio-group/radio-group.component";

@Component({
    selector: "app-radio-groups",
    templateUrl: "./radio-groups.component.html",
})
export class RadioGroupsComponent implements OnInit {
    list: Item[] = [
        { key: "One", value: "1", label: "One" },
        { key: "Two", value: "2", label: "Two" },
    ];
}
```

Reply:
you forgot to remove implements OnInit ;)